### PR TITLE
Add software for Windows Server 2022

### PR DESCRIPTION
**Pull request by kostyanf14**

### CODE REVIEW

File: Test-NETHLK/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# License for the contents of the archive Test-NETHLK.2021.7.28.55.zip
```
Comment (YanVugenfirer) [inline Test-NETHLK/LICENSE.md:1]: Can't you download the license from github together with the script?

Comment (kostyanf14) [reply Test-NETHLK/LICENSE.md:1]: The license is already present in the repository and in the module archive itself. I just thought that I should put a copy into our repository.

Comment (YanVugenfirer) [reply Test-NETHLK/LICENSE.md:1]: I don't think it is needed in such a case. We are downloading the module archive with the license.

Comment (kostyanf14) [reply Test-NETHLK/LICENSE.md]: Removed

**Pull request merged**